src/themes/light/custom.js

```diff
@@ -106,6 +109,9 @@ export const custom = {
       highlightLine: {
           stroke: 'hsla(0, 45%, 90%, 90%);'
       },
+      highlightLineBorder: {
+        stroke:  'hsla(204.762deg, 53%, 46%, 1);'
+      },
```

Comment:
You should not use same color for all 3 style. Have you verified it. 
Please add the snapshot of 3 different style of Lines.

Reply:
Yes sir. Done